packages/editor/src/components/inserter/menu.js

```diff
@@ -378,45 +387,34 @@ export default compose(
 		// Since it's a function only called when the event handlers are called,
 		// it's fine to extract it.
 		// eslint-disable-next-line no-restricted-syntax
-		function getInsertionPoint() {
+		function getInsertionIndex() {
```

Comment:
We should probably also update the `getInsertionPoint` in the comment above.

Reply:
Updated